proof/invariant-abstract/AARCH64/Machine_AI.thy

```diff
@@ -69,6 +69,7 @@ crunch_ignore (no_irq) (add:
   handleE' handleE handle_elseE forM forM_x
   zipWithM ignore_failure)
 
+(* FIXME arch_split: no global_naming? *)
 context Arch begin
```

Comment:
Looks like a potential mistake. Worth trying out what breaks if we change it to `arch_global_naming`. Would also standardise the `requalify` at the end of the file.

Reply:
fixed